torch/csrc/jit/passes/quantization.cpp

```diff
@@ -56,12 +57,100 @@ graph(%self, %input):
     return (%r))";
   std::vector<std::string> patterns = {conv_functional_relu, conv_relu_module};
 
-  std::unordered_set<Value*> values;
   for (const auto& pattern : patterns) {
-    findValuesInPattern(*graph, pattern, values);
+    findValuesInPattern(*graph, pattern, values_to_skip);
+  }
+}
+
+// Propagate current values_to_skip through module hierarchy,
+// adding values to skip in child method graph
+void inputOutputValuesToSkipObserver(
```

Comment:
FYI, this part of code will be deleted in the later diff, so no need to spend too much time reviewing this part